src/tests/ftest/io/ior_single_server.py

```diff
@@ -24,114 +24,116 @@
 from __future__ import print_function
 
 import os
-import json
 
-from apricot import Test
-
-import agent_utils
-import server_utils
+from apricot import TestWithServers
+from daos_api import DaosPool
+from ior_utils import IorCommand, IorFailed
+from mpio_utils import MpioUtils
 import write_host_file
-import ior_utils
-from daos_api import DaosContext, DaosPool, DaosApiError
 
-class IorSingleServer(Test):
-    """
-    Tests IOR with Single Server config.
+
+class IorTestBase(TestWithServers):
```

Comment:
Should this be extracted out of being part of one of the many IOR tests into a utility module in a similar fashion to https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/daos_core_base.py.

Reply:
Agreed, I will implement the change